components/containers/provider-dashboard/helpers/checkSocialMediaValidation.ts

```diff
@@ -39,7 +39,7 @@ export const checkSocialMediaValidation = (
   email: string | null,
   telegram: string | null
 ) => {
-  const emailValidation = email ? validateEmail(email) : true;
+  const emailValidation = email ? validateEmail(email) : false;
```

Comment:
use better naming for emailValidation.
the variable is boolean so the name of the variable can be `isEmailVerified`

Reply:
Done!